src/components/ImageList/index.jsx

```diff
@@ -1,20 +1,36 @@
+import React, { memo } from 'react';
 import { ASYNC_STATUS } from "../../constants/LoadingStates";
 import { useAppContext } from "../../context/appContext";
 import { useAsync } from "../../hooks/useAsync";
 import { LoadingSpinner } from "../LoadingSpinner";
+import { UserAvatar } from "../../components";
 import './ImageList.scss'
-export const ImageList = () => {
+export const ImageList = memo(() => {
 	const { appState, loadImageList } = useAppContext()
 	const { imageList, appStatus } = appState
 	const isAppLoading = appStatus === ASYNC_STATUS.PENDING
 
-	useAsync({asyncFunction: loadImageList})
+	const displayAvatar = (user) => {
+		if(user !== undefined) {
```

Comment:
```suggestion
		if(!user) {
```
Using `!== undefined` is very explicit and doesn't catch other falsey cases. This sometimes is intentional (we have a few cases of it in our repo even!) but in your case here, if user is `null` (ie, the key exists but the value does not), it'll end up failing to render later down in your code

Always be cautious with how you handle falseys - a blank string and the number 0 are falsey too, so always be intentional about when you use explicit checks (like your !== undefined here) and when you use falsey checks (like my !user suggestion)